src/libponyc/codegen/genexe.c

```diff
@@ -294,7 +294,7 @@ static bool link_exe(compile_t* c, ast_t* program,
 
   snprintf(ld_cmd, ld_len,
     "%s -execute -no_pie -arch %.*s "
-    "-macosx_version_min 10.12 -o %s %s %s %s "
+    "-macosx_version_min 11.0.0 -o %s %s %s %s "
```

Comment:
see previous above this version.

Reply:
The `-no_pie` option is not supported on the M1, so we should also include it only on Apple ARM.